.gitignore

```diff
@@ -32,4 +32,6 @@ valueset-db.js
 
 # vscode
 /.vscode
+/data
 .env
+
```

Comment:
Should we add strides.js to gitignore, so that converted data is not pushed, and we keep the empty `stridesData` object as the one that is checked in to strides.js?

Reply:
The challenge is that the JavaScript module bundler (like Webpack) used in build process attempts to resolve all import statements at compile time. If the file doesn't exist at that point, it results in a compilation error, even though the dynamic import is designed to handle such cases at runtime.

Reply:
This was addressed by the most recent group of commits, through a prebuild script to generate the strides.js file! See commit: https://github.com/ccsm-cds-tools/ccsm-cds-dashboard/pull/24/commits/ce6e3accc1d9d4667c1b59b95dac1b720b05d540.